src/components/ClusterManager.js

```diff
@@ -502,6 +510,13 @@ export default ajaxCaller(class ClusterManager extends PureComponent {
         case 'Stopping':
         case 'Creating':
```

Comment:
Note that while a cluster is creating, if you hover your mouse over the "ClusterIcon 'sync'" that your mouse cursor is a click-hand, not a pointer. It should be a pointer because clicking on the 'sync' does nothing.